### PR TITLE
reco: change default timeout to 3 mins

### PR DESCRIPTION
**Pull request by GalNakash-RecoLabs**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by GalNakash-RecoLabs** [comment]
@Shellyber

**Review by amshamah419 — APPROVED** [review]

File: Packs/Reco/Integrations/Reco/Reco.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
 DEMISTO_OCCURRED_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
 RECO_TIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
 DEMISTO_INFORMATIONAL = 0.5
-RECO_API_TIMEOUT_IN_SECONDS = 30  # Increase timeout for RECO API
+RECO_API_TIMEOUT_IN_SECONDS = 180  # Increase timeout for RECO API
```
Comment (amshamah419) [inline Packs/Reco/Integrations/Reco/Reco.py:29]:
```suggestion
RECO_API_TIMEOUT_IN_SECONDS = 180
```

**Comment by amshamah419** [comment]
Hi @GalNakash-RecoLabs we were having some difficulties with the CLA bot, any chance you could resign?

**Comment by GalNakash-RecoLabs** [comment]
RE-SIGN WHAT?

**Comment by amshamah419** [comment]
@GalNakash-RecoLabs It should be here - https://cla-assistant.io/demisto/content?pullRequest=37464 It's the contributor agreement. I think when the bot broke, it lost the fact that you've already done it awhile back.

**Pull request merged**